ktor-client/ktor-client-tests/ktor-client-tests-dispatcher/macosX64/src/TestMacosX64.kt

```diff
@@ -17,7 +17,7 @@ actual fun testSuspend(
 
     val task = launch { block() }
     while (!task.isCompleted) {
-        val date = NSDate().addTimeInterval(1.0) as NSDate
+        val date = NSDate().addTimeInterval(0.01) as NSDate
```

Comment:
Magic -> Magic change :). Let's extract it a named `const`

Reply:
I added `TIME_QUANTUM` const, please have a look.